### PR TITLE
Ingest Budgets v2 release notes

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to create a release note for Ingest budget V1 deprecation.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-service/2024-06-22-manage.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Ingest Budgets (Manage)
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:2]:
```suggestion
title: Deprecation of Legacy Ingest Budgets V1 – Action Required by September 9, 2024 (Manage) 
```

File: blog-service/2024-06-22-manage.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+keywords:
+  - ingest budgets
+  - manage
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+In November 2020, Sumo Logic announced the release of a product enhancement to our Ingest Budget feature (v2) that allowed greater flexibility and granularity in the metadata and fields, which is defined as part of an Ingest Budget Definition. Since then, when it was not possible to create a new Ingest Budget in v2, existing v1 budgets continued to operate in the way that they were defined.
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:13]:
```suggestion
In November 2020, we introduced Ingest Budgets V2, offering greater flexibility and granularity in metadata and fields within the Ingest Budget Definition. Despite this update, existing V1 budgets continued to function as originally defined when creating new V2 budgets was not possible.
```

File: blog-service/2024-06-22-manage.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+**On September 9. 2024**, Sumo Logic will deprecate and remove the existing v1 Ingest Budgets that remain defined in customer Instances.
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:15]:
```suggestion
**On September 9. 2024**, Sumo Logic will deprecate and remove the existing V1 Ingest Budgets that remain defined in customer Instances. 
```

File: blog-service/2024-06-22-manage.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+In November 2020, Sumo Logic announced the release of a product enhancement to our Ingest Budget feature (v2) that allowed greater flexibility and granularity in the metadata and fields, which is defined as part of an Ingest Budget Definition. Since then, when it was not possible to create a new Ingest Budget in v2, existing v1 budgets continued to operate in the way that they were defined.
+
+**On September 9. 2024**, Sumo Logic will deprecate and remove the existing v1 Ingest Budgets that remain defined in customer Instances. 
+
+Prior to September 9, 2024, review your current Ingest Budgets located in **Manage Data > Collection > Ingest Budgets** and any budgets tagged as v1. If these budgets are still needed and valid, you can [edit them and provide a scope to upgrade them to v2](/docs/manage/ingestion-volume/ingest-budgets/daily-volume/#versions).
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:17]:
```suggestion
To ensure uninterrupted service, please review your current Ingest Budgets under **Manage Data > Collection > Ingest Budgets**. Identify any budgets tagged as V1 and, if still needed, [edit and upgrade them to V2](https://github.com/docs/manage/ingestion-volume/ingest-budgets/daily-volume/#versions) before the deprecation date.
```

File: blog-service/2024-06-22-manage.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+:::note
+After the deprecation date, these legacy Ingest Budgets will be automatically removed, and if not migrated you can no longer limit ingestion.
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:20]:
```suggestion
After September 9, 2024, any legacy Ingest Budgets that are not migrated will be automatically removed, resulting in the loss of ingestion limitation functionality.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
a few changes!

**Review by kimsauce — APPROVED** [review]

**Review by jschwegler-sumo — CHANGES_REQUESTED** [review]
**Update this sentence:**
continued to function as originally defined when creating new V2 budgets was not possible.

**With:**
continued to function as originally defined while only allowing v2 budgets for all new configurations going forward.

**Can we also just uncapitalize instances here?**

Ingest Budgets that remain defined in customer Instances

File: blog-service/2024-06-22-manage.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+keywords:
+  - ingest budgets
+  - manage
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+In November 2020, we introduced Ingest Budgets V2, offering greater flexibility and granularity in metadata and fields within the Ingest Budget Definition. Despite this update, existing V1 budgets continued to function as originally defined when creating new V2 budgets was not possible.
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:13]:
```suggestion
In November 2020, we introduced Ingest Budgets V2, offering greater flexibility and granularity in metadata and fields within the Ingest Budget Definition. Despite this update, existing V1 budgets continued to function as originally defined while only allowing v2 budgets for all new configurations going forward.
```

File: blog-service/2024-06-22-manage.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+**On September 9. 2024**, Sumo Logic will deprecate and remove the existing V1 Ingest Budgets that remain defined in customer Instances. 
+
+To ensure uninterrupted service, review your current Ingest Budgets under **Manage Data > Collection > Ingest Budgets**. Identify any budgets tagged as V1 and, if still needed, [edit and upgrade them to V2](https://github.com/docs/manage/ingestion-volume/ingest-budgets/daily-volume/#versions) before the deprecation date.
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:17]:
```suggestion
To ensure uninterrupted service, review your current ingest budgets under **Manage Data > Collection > Ingest Budgets**. Identify any budgets tagged as V1 and, if still needed, [edit and upgrade them to V2](https://github.com/docs/manage/ingestion-volume/ingest-budgets/daily-volume/#versions) before the deprecation date.
```

File: blog-service/2024-06-22-manage.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+:::note
+After September 9, 2024, any legacy Ingest Budgets that are not migrated will be automatically removed, resulting in the loss of ingestion limitation functionality.
```
Comment (kimsauce) [inline blog-service/2024-06-22-manage.md:20]:
```suggestion
After September 9, 2024, any legacy ingest budgets that are not migrated will be automatically removed, resulting in the loss of ingestion limitation functionality.
```

**Comment by kimsauce** [comment]
> **Update this sentence:** continued to function as originally defined when creating new V2 budgets was not possible.
> 
> **With:** continued to function as originally defined while only allowing v2 budgets for all new configurations going forward.
> 
> **Can we also just uncapitalize instances here?**
> 
> Ingest Budgets that remain defined in customer Instances

@jschwegler-sumo , done!

**Comment by jschwegler-sumo** [comment]
> > **Update this sentence:** continued to function as originally defined when creating new V2 budgets was not possible.
> > **With:** continued to function as originally defined while only allowing v2 budgets for all new configurations going forward.
> > **Can we also just uncapitalize instances here?**
> > Ingest Budgets that remain defined in customer Instances
> 
> @jschwegler-sumo , done!

@kimsauce All good from my side.

**Pull request merged**